securesystemslib/util.py

```diff
@@ -77,15 +89,15 @@ def get_file_details(filepath, hash_algorithms=['sha256']):
     raise securesystemslib.exceptions.Error('Path ' + repr(filepath) + ' doest'
         ' not exist.')
 
-  filepath = os.path.abspath(filepath)
```

Comment:
Is this handled by the FilesystemBackend?

Reply:
It isn't handled by the FilesystemBackend. This would probably be OK to leave in, as `abspath()` doesn't access the underlying files or directories.

We don't do this kind of normalisation everywhere we take paths, thus my thought process when making this change was to try and move towards expecting normalised paths in securesystemslib functions. That's probably better as a separate PR and discussion, though.

I'll force push a change to add this back.